sdk/keyvault/Azure.Security.KeyVault.Keys/tests/KeysTestBase.cs

```diff
@@ -284,7 +285,7 @@ protected Task WaitForPurgedKey(string name)
                     {
                         return (Response)null;
                     }
-                }, delay: PollingInterval);
+                }, delay: delay.Value);
```

Comment:
I considered that - and it's already 20 (default) - but then you hammer the server more and it has a low rate limit so I thought that could cause further issues (though, I don't know if failed requests count against the rate limit). I figured this way we space our the requests a little more, but not too much. If we continue to need to bump this out I could increase the max iterations then since 5s polling delay should be more than enough in general.